boards/arm/npcx7m6fb_evb/doc/index.rst

```diff
@@ -0,0 +1,98 @@
+.. _npcx7m6fb_evb:
+
+NPCX7M6FB_EVB
+###################
+
+Overview
+********
+
+The NPCX7M6FB_EVB kit is a development platform to evaluate the
+Nuvoton NPCX7 series microcontrollers. This board needs to be mated with
+part number NPCX796FB.
+
+Hardware
+********
+
+- ARM Cortex-M4F Processor
+- 256 KB RAM and 64 KB boot ROM
+- ADC & GPIO headers
+- UART0 and UART1
+- FAN PWM interface
+- Jtag interface
+- MECC headers
+
+For more information about the SOC please visit:
+
+- `NPCX796FB Reference Manual`_
```

Comment:
Is this  link missing below?

Reply:
Oops. We don't provide 'npcx chip reference manual' on the website. I will remove this line.